cmake/ci.cmake

```diff
@@ -343,6 +349,7 @@ set(GCC_CXXFLAGS
     -Wstrict-aliasing
     -Wstrict-aliasing=3
     -Wstrict-null-sentinel
+    -Wno-strict-overflow
```

Comment:
We just removed this one. When you fix the duplication issue, you should also add a blacklist.